src/Components/Web.JS/src/Platform/Circuits/CircuitStartOptions.ts

```diff
@@ -49,6 +49,8 @@ export interface ReconnectionHandler {
   onConnectionUp(): void;
 }
 
+// eslint-disable-next-line array-element-newline, array-bracket-newline
+const defaultRetryIntervalsSeconds = [1, 3, 5, 10, 15, 30];
```

Comment:
Why not try the first reconnect attempt immediately?

```suggestion
const defaultRetryIntervalsSeconds = [0, 1, 3, 5, 10, 15, 30];
```